src/core/xml_parser.js

```diff
@@ -328,6 +328,10 @@ class SimpleDOMNode {
       .join("");
   }
 
+  get children() {
+    return this.childNodes;
```

Comment:
Given the method just below, it seems that `this.childNodes` could *possibly* be undefined in the general case.
Hence it seems like a good idea to prevent run-time errors elsewhere, by doing:
```javascript
return this.childNodes || [];
```